AdrianBarros/assignments/slidingwindow/lc209/lc209.py

```diff
@@ -1,45 +1,62 @@
 import math
 
+# inputs: int= s arr=[]
+# output: int=n
+# loop throu the arr
+# window start, window end
+# des> i = slide the window
+# until the sum > des
+# min num_i
+
```

Comment:
Good work on writing some high level overview, you're almost there! Make sure that the high level overview is a tad bit more of a short story. It does not have to be as excessive as the example that I gave in slack but somewhere between what you have done and what I have done. #KeepItGoing